products/BellHybrid/apps/application-bell-main/ApplicationBellMain.cpp

```diff
@@ -33,6 +40,13 @@ namespace app
         : Application(name, parent, statusIndicators, startInBackground, stackDepth)
     {
         bus.channels.push_back(sys::BusChannel::ServiceDBNotifications);
+        bus.channels.push_back(sys::BusChannel::LayoutChangeRequests);
+
+        connect(typeid(gui::ChangeHomeScreenLayoutMessage), [&](sys::Message *msg) {
```

Comment:
and where is it sent?

Reply:
It is only "backend" for homescreen changes. Setting those is split to BH-1314

Reply:
thanks. From my perspective, it would make more sense to give a more complete solution even if the PR would be bigger. (or split it in smaller functional chunks)